Yeti-experience/draft-song-yeti-testbed-experience-02.xml

```diff
@@ -392,13 +407,14 @@ SERIAL will trigger the operation of adding these changes to Yeti root zone.</t>
       
       <section title="Yeti Root Servers">
 
-      <t> In Yeti Root system, authoritative servers donated and operated by
+      <t> In Yeti root system, authoritative servers donated and operated by
    Yeti volunteers are configured as a slave to the Yeti DM.  As the
-   time of writing, there are 14 Yeti root servers distributed around
-   the world.  As one of operational research goal, all authoritative
-   servers are required to work in an IPv6-only environment. In addition,
-   until the IANA root, Yeti root server only serve the Yeti root zone, 
-   other than root-servers.org zone and .arpa zone.</t>
+   time of writing, there are 15 Yeti root servers distributed around
+   the world.  It is expected 25 root servers can be supported in Yeti considering 
+   the packet size of priming response within IPv6 MTU 1280 Octets. As one of 
```

Comment:
Do we have any actual research to show that 25 can fit within 1280? With our current setup, it depends on the particular names used, right?

Also, the current root servers do not fit in the DNS size of 512 bytes, and have not since the first IPv6 address was added, so I am not sure why any particular boundary is all that important.